modules/data-table/test/spec.coffee

```diff
@@ -1858,101 +2063,6 @@ describe 'data-table', ->
           setTimeout(f, 50)
 
 
-  describe 'url feed', ->
```

Comment:
Restructured the tests a little - this is now nested in the 'data feeds' section (where it was probably supposed to be from the off)